apbs/chemistry/atom_complex_calc.py

```diff
@@ -126,13 +122,6 @@ def atom_surface(self, atom: Atom, ref: Surface, prad: float) -> Surface:
                 surf.coords.append((rad * ref.coords[i] + apos))
                 surf.coords[-1].is_on_surf = True
 
-        surf.area = (
-            4.0
-            * math.pi
-            * rad
-            * rad
-            * float(surf.npoints)
-            / float(ref.npoints)
-        )
+        surf.area = 4.0 * math.pi * rad * rad * float(surf.npoints) / float(ref.npoints)
```

Comment:
Should we use `numpy.pi` rather than mixing the `math` and `numpy` modules?

https://numpy.org/doc/stable/reference/constants.html

Reply:
I think an old version must have been pushed -- this one reverted back to math instead of numpy.